zkvm/docs/zkvm-spec.md

```diff
@@ -828,6 +787,39 @@ V == v·B + 0·B2
 1. Prover shows `v`.
 2. Verifier checks equality `V == v·B`.
 
+### Taproot
```

Comment:
Leave a blank line between section headings and the text that follows them. Running this file through the `mdfmt` filter (from the i10r repo) will enforce this and other local style conventions.

Reply:
i'd suggest `mdfmt`-ing the entire doc as a separate PR since this can touch many lines unrelated to the subject.